src/components/__tests__/LongTextField.spec.js

```diff
@@ -0,0 +1,45 @@
+import { shallowMount, mount } from '@vue/test-utils'
+import LongTextField from '../fields/LongTextField.vue'
+
+const exampleProps = {
+  id: 1,
+  name: 'example',
+  required: true,
+  placeholder: 'enter text here',
+  prompt: 'How much do you love this input?',
+}
+
+const helpTextProps = {
+  id: 1,
+  name: 'example',
+  required: true,
+  placeholder: 'enter text here',
+  prompt: 'How much do you love this input?',
+  helpText: 'Help',
+}
+
+describe('match the snapshot', () => {
+  it('renders correctly', () => {
+    const wrapper = shallowMount(LongTextField, { propsData: exampleProps })
+    expect(wrapper.html()).toMatchSnapshot()
+    wrapper.destroy()
+  })
+})
+
+
+describe('help text passed in', () => {
+  it('help text appears when true', () => {
+    const wrapper = mount(LongTextField, { propsData: helpTextProps })
+    expect(wrapper.element.getElementsByTagName('p')[1].textContent.replace(/\s/g, ''))
```

Comment:
[nb] so we don't rely on `helpText` remaining in specifically a  `<p>`, let's just search the text. Also you're welcome to whack it into one `it` instead of the double describes. Also can dry up example props through destructuring.

```suggestion
it('renders helpText only when provided', () => {
    const helpText = 'Help!'
    
    let wrapper =  mount(LongTextField, { propsData: exampleProps })
    expect(wrapper.text()).not.toContain(helpText))
    
    wrapper = mount(LongTextField, { propsData: { helpText, ...exampleProps })
    expect(wrapper.text()).toContain(helpText))
})
```